### PR TITLE
[ticket/14882] Add core event to MCP after move post sync

### PR DESCRIPTION
**Pull request by dsinn**
Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-14882

PHPBB3-14882

### CODE REVIEW

File: phpBB/includes/functions_admin.php [hunk 1]
```diff
@@ -642,7 +642,7 @@ function move_posts($post_ids, $topic_id, $auto_sync = true)
 	 * @event core.move_posts_before
 	 * @var	array	post_ids	Array of post ids to move
 	 * @var	string	topic_id	The topic id the posts are moved to
-	 * @var bool	auto_sync	Whether or not to perform auto sync
```
Comment (CHItA) [inline phpBB/includes/functions_admin.php]: Do you know what these changes are exactly?

Comment (dsinn) [inline phpBB/includes/functions_admin.php]: The file was not consistent with its tab/space characters, so I corrected it.  I'll remember to include that in the commit message next time.

File: phpBB/includes/functions_admin.php [hunk 2]
```diff
@@ -698,6 +698,28 @@ function move_posts($post_ids, $topic_id, $auto_sync = true)
 		sync('topic_attachment', 'topic_id', $topic_ids);
 		sync('topic', 'topic_id', $topic_ids, true);
 		sync('forum', 'forum_id', $forum_ids, true, true);
+
+		/**
+		 * Perform additional actions after move post sync
+		 *
+		 * @event core.move_posts_sync_after
+		 * @var	array	post_ids	Array of the moved post ids
+		 * @var	string	topic_id	The topic id the posts are moved to
```
Comment (marc1706) [inline phpBB/includes/functions_admin.php]: ID should be int

Comment (dsinn) [inline phpBB/includes/functions_admin.php]: This was copied from one of the two move_posts events above, so I've fixed those too.

**Pull request merged**